woocommerce/woocommerce-ios
# Store onboarding: Update payment task description, priority and URL

Closes: #10570 


## Description

This PR updates a few things for the payment onboarding task:
- Prioritize it over the custom domain and launch store tasks.
- New description following 7abnJ5szUDhwiczNwhcyW3-fi
- Updated the URL to open for `woocommerce-payment` task following pe5sF9-1VU-p2.

## Testing instructions

- Log in to the app.
- Open a free trial site or create one if needed.
- Navigate to the My Store screen, notice the payment task on the onboarding task list is above the custom domain and launch store tasks. The description should be correct.
- Tap the task > Continue Setup. A web view should be displayed showing the WooPayments page with the Finish Setup button displayed.

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/e3a964c0-d878-4d0f-a3c6-83b98ab06187" width=320 /> <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/b2d01171-6809-4b46-8f47-a6a01f075c50" width=320 />

---
